internal/processor/mdblist.go

```diff
@@ -82,8 +83,9 @@ func (s Service) mdblist(ctx context.Context, cfg *domain.ListConfig, dryRun boo
 
 		if !dryRun {
 			if err := brr.UpdateFilterByID(ctx, filterID, f); err != nil {
-				l.Error().Err(err).Msgf("something went wrong updating filter: %v", filterID)
-				return fmt.Errorf("error updating filter: %v, %w", filterID, err)
+				errMsg := fmt.Sprintf("error updating filter: %v, %v", filterID, err)
+				l.Error().Msg(errMsg)
```

Comment:
The old log line had the `Error().Err(err)` which gives better error output than doing it all in the `Msg`.